### PR TITLE
Made "share" link more proeminent on album page

### PR DESCRIPTION
**Pull request by LouisLouiseKay**
Now you can "share" right from underneath album image

### CODE REVIEW

File: client/src/components/TrackGroup/TrackGroupEmbed.tsx [hunk 1]
```diff
@@ -87,18 +88,20 @@ const TrackGroupEmbed: React.FC<{
...
-          font-size: 1.2rem;
-          background: transparent;
-          color: var(--mi-primary-color);
-          margin-left: 0.2rem;
-        `}
-        onClick={() => setIsPopupOpen(true)}
-      >
-        <Button>{t("share")}</Button>
-      </li>
+      <div onClick={() => setIsPopupOpen(true)}>
```
Comment (simonv3) [inline client/src/components/TrackGroup/TrackGroupEmbed.tsx:91]: hmm, is there a reason the onClick isn't on the Button?

File: client/src/components/TrackGroup/Wishlist.tsx [hunk 2]
```diff
@@ -28,10 +29,20 @@ const Wishlist: React.FC<{ trackGroup: TrackGroup }> = ({ trackGroup }) => {
   return (
     <Button
       onClick={onClick}
-      compact
-      variant="link"
       className={css`
         color: inherit !important;
+        :hover {
```
Comment (simonv3) [inline client/src/components/TrackGroup/Wishlist.tsx:34]: Does this work? This doesn't have to be &:hover?

```suggestion
        &:hover {
```

Comment (LouisLouiseKay) [reply client/src/components/TrackGroup/Wishlist.tsx:34]: Yeah it worked without the &

File: client/src/components/TrackGroup/TrackGroupEmbed.tsx (hunk 1 above)
Comment (LouisLouiseKay) [reply client/src/components/TrackGroup/TrackGroupEmbed.tsx]: Ok done !

File: client/src/components/TrackGroup/Wishlist.tsx (hunk 2 above)
Comment (simonv3) [reply client/src/components/TrackGroup/Wishlist.tsx:34]: I guess I don't understand that 😬

**Pull request merged**